src/core/webidl.js

```diff
@@ -382,14 +382,9 @@ export async function run() {
   if (!idls.length) {
     return;
   }
-  if (!document.querySelector(".idl:not(pre), .webidl:not(pre)")) {
-    const link = document.querySelector("head link");
-    if (link) {
-      const style = document.createElement("style");
-      style.textContent = css;
-      link.before(style);
-    }
-  }
+  const style = document.createElement("style");
```

Comment:
But shouldn't there be some IDL defined before we add the stylesheet? I wonder what went wrong in that other spec?

Reply:
The check in previous block ensures that. (early return if no `pre.idl, pre.webidl`)

Reply:
Oh, lol. I see now. Yeah, this was a silly bug.